src/Microsoft.Extensions.Caching.Abstractions/MemoryCacheExtensions.cs

```diff
@@ -45,6 +65,10 @@ public static TItem Set<TItem>(this IMemoryCache cache, object key, TItem value)
 
         public static TItem Set<TItem>(this IMemoryCache cache, object key, TItem value, DateTimeOffset absoluteExpiration)
         {
+            if (key == null)
+            {
+                throw new System.ArgumentNullException(nameof(key));
```

Comment:
Add a using for System.

Reply:
I have updated the code and sent a new pull-request and raised a review.